### ISSUE TITLE
Stop installing packages automatically as part of the build

### ISSUE THREAD

**Issue by penberg** [scylladb/scylla#7219]
The `build_rpm.sh` script in JMX, for example, installs packages automatically for tools it needs. The installation requires `sudo`, which can cause the build process from scylla.git to hang on host that does not have the dependencies.

Let's remove the automatic installation, and report an error instead telling people to run `install-dependencies.sh`.

**Comment by penberg** [comment]
The issue was reported by @psarna

**Comment by psarna** [comment]
Even after running `install-dependencies.sh`, running `ninja` hangs, and it yet again looks like a "sudo in the background" issue. Here's last lines printed by `ninja`:
```bash
Conflicts: cassandra
Processing files: scylla-tools-core-666.development-20200911.2d49ded77b.noarch
Provides: scylla-tools-core = 666.development-20200911.2d49ded77b
Requires(rpmlib): rpmlib(CompressedFileNames) <= 3.0.4-1 rpmlib(FileDigests) <= 4.6.0-1 rpmlib(PayloadFilesHavePrefix) <= 4.0-1
Requires: /bin/sh /usr/bin/env
Checking for unpackaged file(s): /usr/lib/rpm/check-files /home/sarna/repo/scylla/tools/java/build/redhat/BUILDROOT/scylla-tools-666.development-20200911.2d49ded77b.x86_64
Wrote: /home/sarna/repo/scylla/tools/java/build/redhat/SRPMS/scylla-tools-666.development-20200911.2d49ded77b.src.rpm
Wrote: /home/sarna/repo/scylla/tools/java/build/redhat/RPMS/noarch/scylla-tools-666.development-20200911.2d49ded77b.noarch.rpm
Wrote: /home/sarna/repo/scylla/tools/java/build/redhat/RPMS/noarch/scylla-tools-core-666.development-20200911.2d49ded77b.noarch.rpm
Executing(%clean): /bin/sh -e /var/tmp/rpm-tmp.iLWQby
+ umask 022
+ cd /home/sarna/repo/scylla/tools/java/build/redhat/BUILD
+ cd scylla-tools
+ /usr/bin/rm -rf /home/sarna/repo/scylla/tools/java/build/redhat/BUILDROOT/scylla-tools-666.development-20200911.2d49ded77b.x86_64
+ RPM_EC=0
++ jobs -p
+ exit 0
```
And here's the hanging request in the background:
```bash
[sarna@localhost scylla-enterprise]$ ps ax | grep sudo
 684736 pts/0    T      0:00 sudo yum install -y devscripts
 685729 pts/3    S+     0:00 grep --color=auto sudo
```
I'm not sure if it's a different issue or not, so I started with posting these logs here. If it's a separate issue and deserves a separate entry on github, let me know.

**Comment by penberg** [comment]
@psarna Looks like a bug in JMX that was already fixed in tools, see `[PATCH scylla-jmx] dist: debian: fix detection of debuild
` on the mailing list.
